ccd-definition/CaseEventToFields/createOrder-PREPARE_FOR_HEARING.json

```diff
@@ -26,6 +26,77 @@
     "PageShowCondition": "orderTypeAndDocument.type=\"BLANK_ORDER\"",
     "ShowSummaryChangeOption": "Y"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createOrder-PREPARE_FOR_HEARING",
+    "CaseFieldID": "epoChildren",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "COMPLEX",
+    "PageID": "EPOChildren",
+    "PageLabel": " ",
+    "PageDisplayOrder": 3,
+    "PageColumnNumber": 1,
+    "PageShowCondition": "orderTypeAndDocument.type=\"EMERGENCY_PROTECTION_ORDER\"",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createOrder-PREPARE_FOR_HEARING",
+    "CaseFieldID": "epoType",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": "EPOType",
+    "PageLabel": " ",
+    "PageDisplayOrder": 4,
+    "PageColumnNumber": 1,
+    "PageShowCondition": "orderTypeAndDocument.type=\"EMERGENCY_PROTECTION_ORDER\"",
+    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/validate-emergency-protection-order/address/mid-event",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createOrder-PREPARE_FOR_HEARING",
+    "CaseFieldID": "epoRemovalAddress",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "OPTIONAL",
```

Comment:
why not use caseEventToComplexType and ccd mandatory fields here?

Reply:
I had difficulty getting this working, I spoke to you briefly about it. Address was not appearing using caseEventToComplexType. I'm guessing because Address itself is a complex type?